client/src/app/components/mission/mission.component.css

```diff
@@ -2,3 +2,26 @@
     display: flex;
     align-items: center;
 }
+
+a {
+  color: coral;
+  font-weight: 600;
+}
+
+.big_picture {
+  height: 90vh;
+  width: 100vw;
+  background-image: url("../../../assets/mission_background_01.jpg");
```

Comment:
size of those pictures is over 2.4MB!!! make it max 100kb, 1024px width (proportional height)